scripts/manage-db.py

```diff
@@ -126,9 +126,7 @@ def extract_active_data(trans, url, dump_location='dump.sql'):
     ))
 
     popen(
-        _strip_multiple_spaces('%s %s dockerflow rules rules_history rules_scheduled_changes rules_scheduled_changes_conditions \
-        rules_scheduled_changes_conditions_history rules_scheduled_changes_signoffs rules_scheduled_changes_signoffs_history \
-        rules_scheduled_changes_history migrate_version  > %s' % (mysql_default_command, database, dump_location))
+        _strip_multiple_spaces('%s %s dockerflow rules rules_history  > %s' % (mysql_default_command, database, dump_location))
```

Comment:
Looks like you removed "migrate_version" as well - please re-add that, it's important!